apps/web/src/pages/Applications/ApplicationSuccessPage/ApplicationSuccessPage.tsx

```diff
@@ -139,39 +138,61 @@ const ApplicationSuccess = ({ application }: ApplicationPageProps) => {
               data-h2-vertical-align="base(top)"
             >
               {intl.formatMessage({
-                defaultMessage:
-                  "Submit an application to ongoing recruitment talent pools.",
-                id: "ZTnze/",
+                defaultMessage: "Browse for other opportunities",
+                id: "nc5GGm",
```

Comment:
Does it still make sense then to link to `/en/browse/pools#ongoingRecruitments` or should it just point to the top of the browse page?

Reply:
That is a good question. I'm not sure how to answer it though 😅 @substrae any thoughts?

Reply:
@vd1992 @esizer 

Yep good catch - just link to `/browse`

Reply:
Thanks! Updated in [remove hash from browse link](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9632/commits/7490534a2fe3b9e61c3be5de954cdc5dd3e9be93)